documentation/modules/appendix_crds.adoc

```diff
@@ -740,13 +740,41 @@ It must have the value `simple` for the type `KafkaAuthorizationSimple`.
 |string array
 |====
 
+[id='type-KafkaAuthorizationOpa-{context}']
+### `KafkaAuthorizationOpa` schema reference
+
+Used in: xref:type-KafkaClusterSpec-{context}[`KafkaClusterSpec`]
+
+include::../api/io.strimzi.api.kafka.model.KafkaAuthorizationOpa.adoc[leveloffset=+1]
+
+The `type` property is a discriminator that distinguishes the use of the type `KafkaAuthorizationOpa` from xref:type-KafkaAuthorizationSimple-{context}[`KafkaAuthorizationSimple`], xref:type-KafkaAuthorizationKeycloak-{context}[`KafkaAuthorizationKeycloak`].
+It must have the value `opa` for the type `KafkaAuthorizationOpa`.
+[options="header"]
+|====
+|Property                     |Description
+|type                  1.2+<.<|Must be `opa`.
+|string
+|url                   1.2+<.<|The URL used to connect to the Open Policy Agent server. The URL has to include the policy which will be queried by the authorizer. This option is required.
+|string
+|allowOnError          1.2+<.<|Defines whether Kafka client should be allowed or denied by default when the authorizer fails to query the Open Policy Agent (for example because it is temporarily unavailable). Defaults to `false` - all actions will be denied.
+|boolean
+|initialCacheCapacity  1.2+<.<|Initial capacity of the local cache used by the authorizer to avoid querying Open Policy Agent for every request Defaults to `5000`.
+|integer
+|maximumCacheSize      1.2+<.<|Maximum capacity of the local cache used by the authorizer to avoid querying Open Policy Agent for every request. Defaults to `50000`.
```

Comment:
```suggestion
|maximumCacheSize      1.2+<.<|Maximum capacity of the local cache used by the authorizer to avoid querying the Open Policy Agent for every request. Defaults to `50000`.
```